### PR TITLE
Ref issue #135 python 2 to python 3 porting

### PR DESCRIPTION
**Pull request by tai271828**
issue #135 includes the failure to run

`nosetests test_builtin.py:TestLocalParallel.test_soln`

besides, TestLocalParallel.test_soln failed because of many issues coupled together.

This is one of the root cause to the failure of TestLocalParallel.test_soln.

I am going to create several PRs to fix TestLocalParallel.test_soln within few weeks. This is the first one.

### CODE REVIEW

**Comment by tai271828** [comment]
The build failure is caused by the old g++ which not supports c++14. g++ supports c++14 is necessary to build the recent merge c++ code, e.g. this PR https://github.com/solvcon/solvcon/pull/155

I failed to build the recent source code as well on Ubuntu Trusty (14.04) and the CI uses a elder distribution 12.04, which uses this version of gcc on our Travis CI:

`gcc (Ubuntu/Linaro 4.6.3-1ubuntu5) 4.6.3`

File: solvcon/case_legacy.py [hunk 1]
```diff
@@ -700,7 +700,7 @@ def _create_workers_remote(self, dealer, nblk):
         if conf.env.mpi:
             info(' (head excluded for MPI)')
         info(':\n')
-        iworker = 0
```
Comment (yungyuc) [inline solvcon/case_legacy.py:703]: Could you avoid the change of spaces in the bugfix?  You can use another issue to track the space changes.

**Comment by yungyuc** [comment]
Travis-CI fails because we don't have a travis.yml inplace, but yes, you will need a C++14 compiler because of PR #155.

File: solvcon/case_legacy.py (hunk 1 above)
Comment (tai271828) [inline solvcon/case_legacy.py:703]: ok

**Comment by tai271828** [comment]
Updated the source to separate the trailing space change and the real fix. Please review it again.

**Comment by yungyuc** [comment]
This is old.  I'd like to close it.

**Pull request closed without merging**